.env

```diff
@@ -41,4 +41,8 @@ FACEBOOK_APP_SECRET=facebook_app_secret
 
 GOOGLE_APP_ID=id.apps.googleusercontent.com
 
+APPLE_PRIVATE_KEY=private_key
 APPLE_CLIENT_ID=org.cru.godtools
```

Comment:
Hey @andrewroth will APPLE_CLIENT_ID and APPLE_REDIRECT_URI need to be dynamic here depending on ```staging``` vs ```production``` environment?  Or is PR for production only?

Reply:
@levieggertcru for our rails apps the environment variables are managed via a `secrets` command line utility and actually get set as environment variables in the Docker container when it is launched in our AWS infrastructure

Reply:
Thanks @frett makes sense.

Reply:
Right, what @frett said :)